i18n/zh-CN/docusaurus-plugin-content-docs/current/feature/spark-etl.md

```diff
@@ -0,0 +1,443 @@
+---
+title: 支持spark数据同步
+sidebar_position: 0.2
+--- 
+
+## 1. 背景
+用户可以通过配置json的方式进行spark数据同步
+
+## 2. 使用说明
+常见的数据同步示例如下:
+
```

Comment:
batter to add information about:
- list of supported types
- perequisite requirements  for each type
-description of key parameters for each type